server/controllers/v1/comment.ts

```diff
@@ -0,0 +1,218 @@
+import { Comment } from '../../models/comment';
+import db from '../../models';
+
+// The return type of a Comment associated with the Post's User.
+export type CommentsUser = Comment & {
+  User: {
```

Comment:
Check in postman to make sure this type is correct. I remember we had a similar mistake for PostUser, it was returning `UserId` separately from User